packages/components/src/placeholder/index.js

```diff
@@ -27,7 +32,7 @@ function Placeholder( { icon, children, label, instructions, className, notices,
 				</div>
 			}
 			<div className="components-placeholder__label">
-				{ isString( icon ) ? <Dashicon icon={ icon } /> : icon }
+				{ isString( icon ) ? <Dashicon icon={ icon } /> : cloneElement( icon, { height: 24, width: 24 } ) }
```

Comment:
It needs more work as it fails some unit tests:
https://travis-ci.com/WordPress/gutenberg/jobs/261031208

```
Placeholder › basic rendering › should by default render label section and fieldset.
614
615    Invariant Violation: React.cloneElement(...): The argument must be a React element, but you passed undefined.
```

Reply:
It seems like we keep repeating these checks (isString, size...) in all components accepting icons. I wonder if it's better if we just consistently use `<Icon` component (probably needs to be adapted) no matter the "type" of the icon passed.

Reply:
I forgot we have the `Icon` component. I will refactor to use it. Anyway, I'd rather find a quick fix here and tackle it separately as it's just a minor UI glitch :)

Reply:
It looks like `Icon` solves all those issues without any modifications 👍